dotnetcoreVolunteer/Models/User.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+/// <summary>
+/// Summary description for Class1
+/// </summary>
+public class User
+{
+    [Key]
+    public Guid UserId { get; set; }
```

Comment:
All these props shouldn't be ints.  This PR is still fine and changing them will fix them in the next migration.
Just be aware